xmltodict.py

```diff
@@ -392,7 +394,10 @@ def unparse(input_dict, output=None, encoding='utf-8', full_document=True,
     if output is None:
         output = StringIO()
         must_return = True
-    content_handler = XMLGenerator(output, encoding)
+    if pyver[0] > 3 or (pyver[0] == 3 and pyver[1] >= 2):
```

Comment:
Don't compare to the version number, just use the new option and let it fail. This way, if someone is trying to use this option in an unsupported version of Python, it will fail on them explicitly instead of silently ignoring it.

```python
if short_empty_elements:
    content_handler = XMLGenerator(output, encoding, True)
else:
    content_handler = XMLGenerator(output, encoding)
```